Automattic/wp-calypso
# Automattic for Agencies: Update favicons for the new environment

Resolves https://github.com/Automattic/jetpack-genesis/issues/218

## Proposed Changes

This PR updates the favicons for the new environment.

## Testing Instructions


Note: None of the links work on the masterbar. So, please ignore it for now

- Switch to the branch `update/favicon-for-a4a`.
- Start the server by running `start-a8c-for-agencies`.
- Verify that the favicons are correct and the right icon is shown for dev. We cannot verify for staging & production yet, just making sure the right icon is added should be enough for now.

## Pre-merge Checklist
